src/cysignals/signals.pyx

```diff
@@ -108,16 +107,29 @@ cdef public int sig_raise_exception "sig_raise_exception"(int sig, const char* m
         if msg is NULL:
             msg = "Floating point exception"
         PyErr_SetString(FloatingPointError, msg)
-    elif sig == SIGBUS:
-        if msg is NULL:
-            msg = "Bus error"
-        PyErr_SetString(SignalError, msg)
     elif sig == SIGSEGV:
         if msg is NULL:
             msg = "Segmentation fault"
         PyErr_SetString(SignalError, msg)
-    else:
-        raise SystemError(f"unknown signal number {sig}")
+    IF UNAME_SYSNAME != 'Windows':
```

Comment:
No need for that.

Reply:
Can you please be more explicit on this one ?

Reply:
I am just saying that you should try to replace the code
```
IF UNAME_SYSNAME == 'Windows':
    X
ELSE:
    Y
```
by
```
X
Y
```
(i.e. remove the conditions)